templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+<p>Are you relly want to delete Manufacturer: <strong class="larger-text">{{ manufacturer }}</strong> ?</p>
+
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>
+      <a
+        href="{% url 'taxi:manufacturer-list' %}"
+    > <input class="btn btn-primary" type="submit" value="CANCEL">

```

Comment:
Do not break the tag in this way, place arguments on the same line with that tag.